src/audio-worklet/free-queue/src/interface/free_queue.h

```diff
@@ -167,34 +172,34 @@ void* get_free_queue_pointers(struct FreeQueue* fq, char* data) {
 /**
  * Helper function for debugging
 */
-EMSCRIPTEN_KEEPALIVE void print_data(struct FreeQueue* fq) {
-    for (uint32_t channel = 0; channel < fq->channelCount; channel++) {
+EMSCRIPTEN_KEEPALIVE void print_data(struct FreeQueue* queue) {
+    for (uint32_t channel = 0; channel < queue->channelCount; channel++) {
         printf("channel %d: ", channel);
-        for (uint32_t i = 0; i < fq->bufferLength; i++) {
-            printf("%f ", fq->channels[channel][i]);
+        for (uint32_t i = 0; i < queue->bufferLength; i++) {
+            printf("%f ", queue->channelData[channel][i]);
         }
         printf("\n");
     }
 
-    uint32_t currentRead = atomic_load(fq->state + READ);
-    uint32_t currentWrite = atomic_load(fq->state + WRITE);
+    uint32_t currentRead = atomic_load(queue->state + READ);
+    uint32_t currentWrite = atomic_load(queue->state + WRITE);
 
     printf("----------\n");
     printf("currentRead: %u  | currentWrite: %u\n", currentRead, currentWrite);
-    printf("availabeRead: %u  | availableWrite: %u\n", _getAvailableRead(fq, currentRead, currentWrite), _getAvailableWrite(fq, currentRead, currentWrite));
+    printf("availabeRead: %u  | availableWrite: %u\n", _getAvailableRead(queue, currentRead, currentWrite), _getAvailableWrite(queue, currentRead, currentWrite));
     printf("----------\n");
 }
 
 /**
```

Comment:
The comment between L193-L195 is same with the one at L172-L174.

Please provide a function overview. What does it do? When should I use it?